src/cpp/database/entities.hpp

```diff
@@ -315,7 +324,7 @@ struct Topic : Entity
             std::string topic_name,
             std::string topic_type,
             std::shared_ptr<Domain> topic_domain) noexcept
-        : Entity(EntityKind::TOPIC, topic_name)
+        : Entity(EntityKind::TOPIC, topic_name, true, is_metatraffic_topic(topic_name))
```

Comment:
be careful with that `true` in the call. 
The correct way would be to change the order of the arguments, check if this will break the API (if it is public, I don't think so).

In the other hand, it could be that when creating an entity, it is always alive. I this case the argument in the Entity constructor must be removed (or create a constructor without it)